Doc/faq/design.rst

```diff
@@ -290,6 +292,9 @@ It's suggested that you use a prefix for the method names, such as ``visit_`` in
 this example.  Without such a prefix, if values are coming from an untrusted
 source, an attacker would be able to call any method on your object.
 
+Imitating switch with fallthrough, as with Fortran's computed GOTO
+and C's switch-case-default, is possible, much harder, and less needed.
```

Comment:
```suggestion
Imitating switch with fallthrough, as with C's switch-case-default,
is possible, much harder, and less needed.
```
While github sees 'fallthrough' as a misspelling, it is at least as common as 'fall-through' and is even used as a declaration in C23.